source/js/components/validation.js

```diff
@@ -0,0 +1,48 @@
+// constants for "Other Amount" inputs and error messages
+const oneTimeOtherAmountInput = document.querySelector(
+  ".one-time-amount-other-input"
+);
+const monthlyOtherAmountInput = document.querySelector(
+  ".monthly-amount-other-input"
+);
+const oneTimeOtherAmountErrorMessage = document.querySelector(
+  ".error-message__one-time-other-amount"
+);
+const monthlyOtherAmountErrorMessage = document.querySelector(
+  ".error-message__monthly-other-amount"
+);
+
+// Handler for "other amount" inputs
+function otherAmountInputValidation() {
+  oneTimeOtherAmountInput.addEventListener("blur", (e) => {
+    if (document.querySelector(".one-time-donation-amount-radio:checked")) {
+      inputValueCheck(oneTimeOtherAmountInput, oneTimeOtherAmountErrorMessage);
+    }
+  });
+  monthlyOtherAmountInput.addEventListener("blur", (e) => {
+    if (document.querySelector(".monthly-donation-amount-radio:checked")) {
+      inputValueCheck(monthlyOtherAmountInput, monthlyOtherAmountErrorMessage);
+    }
+  });
+}
+
+function inputValueCheck(inputObjectToValidate, messageToUpdate) {
```

Comment:
Updated this function to now receive the input object which is being updated, take the minimum/maximum value for that input, and validate that the attempted value is in between those parameters.